### DISCUSSION TITLE
Assignment 2: 1 Juypter Notebook?

### DISCUSSION THREAD

**Discussion by kaiserog** [irgroup-classrooms/dis08-2021#71] [Q&A] [answered]
Guten Abend,

Möchten Sie eine Datei oder sind auch drei Notebooks in Ordnung, d.h. je Aufgabe 1 Jupyter Notebook?

Beste Grüße,
Konstantin Kaiser

**Comment by phschaer** [comment, marked as answer]
Gerne alles in einem Notebook zusammen!
